lib/status/status.go

```diff
@@ -41,6 +41,26 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// NodeStatuses returns the node statuses.
+func NodeStatuses(ctx context.Context) (nodes []ClusterServer, err error) {
```

Comment:
We have `FromPlanetAgent` method below that also returns all nodes and their statuses so I think this extra function is not really needed. You can just use `status.FromPlanetAgent(ctx, nil)` like we do in other places.